pdns/recursordist/docs/changelog/4.1.rst

```diff
@@ -1,6 +1,57 @@
 Changelogs for 4.1.x
 ====================
 
+.. changelog::
+  :version: 4.1.2
+  :released: Unreleased
+
+  This releases improves the stability and resilency of the RPZ implementation and fixes several issues related to EDNS Client Subnet.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 6298, 6303, 6290, 6268
+
+    Add the option to set the AXFR timeout for RPZs.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 6336, 6237, 6293
+    :tickets: 6238
+
+    Retry loading RPZ zones from server when they fail initially.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 6172
+
+    IXFR: correct behavior of dealing with DNS Name with multiple records and speed up IXFR transaction (Leon Xu).
```

Comment:
The description is not quite right for the rec, only the speed up part is relevant there.